_posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md

```diff
@@ -0,0 +1,119 @@
+
+# How to use Presto with AWS efficiently
+
+We could learn how to use Presto with AWS including Presto on EMR, Presto on EC2, Presto by Athena and AWS Glue.
+Noritaka Sekiyama, Sr. Big Data Architect at Amazon Web Service, Japan, also shares a comparison of Presto on AWS (EC2, EMR, Athena). 
+If you are a new to Presto, his talk gives you an insight to choose your first Presto environement.
+
+<iframe src="//www.slideshare.net/slideshow/embed_code/key/GR6e3dfKKJ8w4c" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe> <div style="margin-bottom:5px"> <strong> <a href="//www.slideshare.net/SatoruKamikaseda/support-presto-as-a-feature-of-saas" title="Support Presto as a feature of SaaS" target="_blank">Support Presto as a feature of SaaS</a> </strong> from <strong><a href="https://www.slideshare.net/SatoruKamikaseda" target="_blank">SatoruKamikaseda</a></strong> </div>
+
+# Presto @ LINE 2020
+
+LINE is the biggest company providing the mobile communication tool in Japan (say WhatsApp in Japan). HYuya Ebihara, one of Presto mainteners, 
+gives us how they improve Presto at their platform since they presented in the previous conference (https://prestosql.io/blog/2019/07/11/report-for-presto-conference-tokyo.html). 
+Their Presto usage significantly increases from 2019. Num of Presto workers from 100 to 300 and Num of daily queries reaches to 50,000 queries from 20,000 queries. 
+We could learn how to upgrade Presto from 314 to 339 and how they resolved issues through Presto upgrade.
+
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vS2QdQjhLsiSuVdWlEmT23ixqoZXkRrKKMRGa1hrZHg65OpcH18RpzARotOMYvIBSwP57lPPAHkUQOx/embed" frameborder="0" width="1280" height="749" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+
+# Dive into Amazon Athena - Serverless Presto, 2020
+
+Makoto Kawamura, Solution Architect at Amazon Web Service Japan, 
+introduces the latest features of AWS Athena and performance tuning tips. It must be helpful for developers who tied to AWS to explore Amazon Athena.
+
+<script async class="speakerdeck-embed" data-id="92a399aad5344df197279cd4195d9464" data-ratio="1.77777777777778" src="//speakerdeck.com/assets/embed.js"></script>
+
+# Presto Cassandra Connector Hack at Repro
+
+Repro provides Customer Engagement Platform that enables companies to personalize their communication strategies with the right message at the right time to drive better retention and lifetime value. 
+They use Presto for a segmentation backend system in their service to make a list of audiences with a certain condition. 
+
+![](https://docs.repro.io/en/_images/segmentation-filter.png)
```

Comment:
Better not use an external image like that .. maybe add it to the PR in a suitable size or link to it.

Reply:
let me remove it.